Python/Product/Analysis/Parsing/Parser.cs

```diff
@@ -76,14 +76,14 @@ public class Parser : IDisposable { // TODO: remove IDisposable
             _langVersion = langVersion;
             _verbatim = verbatim;
             _bindReferences = bindRefs;
-            
+
```

Comment:
This is the actual fix, _languageFeatures needs to be assigned before we reset it